.github/workflows/build_cmake.yml

```diff
@@ -116,7 +116,7 @@ jobs:
     - name: Tar Linux Server Binaries
       if: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/releases')) && (runner.os == 'Linux') }}
       run: >-
-        tar -cvzf ni-grpc-device-server-linux-glibc2_23-x64.tar.gz
+        tar -cvzf ni-grpc-device-server-linux-glibc2_27-x64.tar.gz
```

Comment:
Definitely a choice I made previously that was short-sighted, but we should probably consider storing the glibc version in a variable to make updating easier in the future.